OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -117,16 +118,13 @@ public bool Disabled
 					return;
 
 				disabled = value;
-				Invalidate(map.ProjectedCellBounds);
 			}
 		}
 
 		bool fogEnabled;
 		public bool FogEnabled { get { return !Disabled && fogEnabled; } }
 		public bool ExploreMapEnabled { get; private set; }
 
-		public int Hash { get; private set; }
```

Comment:
This is an important safety net for diagnosing desyncs, can we find a way to keep this?

Reply:
Restored.